platform/ios/PolyPodApp/PodApi/PostOffice.swift

```diff
@@ -201,11 +201,9 @@ extension PostOffice {
         
         PodApi.shared.polyIn.hasQuads(quads: extendedDataSet) { doesHave in
             if doesHave {
-                completionHandler(MessagePackValue(), true)
+                completionHandler(MessagePackValue(true), nil)
             } else {
-                // TODO: This needs a closer look - should we really fail if the
-                //       data we check for is not there?
-                completionHandler(false, createErrorResponse(#function, "Not there"))
+                completionHandler(MessagePackValue(false), nil)
             }
```

Comment:
I guess this can be just:
```
completionHandler(doesHave, nil)
```
`MessagePackValue` conforms to `ExpressibleByBooleanLiteral`, so there is no need to wrap the boolean in `MessagePackValue`

Reply:
That's neat!

Reply:
I tried to pass just a boolean to it, but I get this error:
<img width="700" alt="Screenshot 2022-05-04 at 10 30 07" src="https://user-images.githubusercontent.com/25011107/166639057-23229ea7-a946-4ef1-aa4f-a54544f301a6.png">

So I just wrapped it.

Reply:
Hmm, interesting. `completionHandler(true, nil)` works. Seems to be a swift quirk...